projects/hipfft/clients/tests/CMakeLists.txt

```
@@ -57,8 +57,31 @@ find_package( Boost REQUIRED)
 
 set( Boost_USE_STATIC_LIBS OFF )
 
-
+# searching FFTW with FindFFTW.cmake that is provided by project
+# because distro versions of FFTW does not include the cmake specific
+# helper files
 find_package( FFTW 3.0 REQUIRED MODULE COMPONENTS FLOAT DOUBLE )
+if( FFTW_FOUND )
+  message(STATUS "FFTW found by FindFFTW.cmake")
+else()
+  # searching fftw versions build from source code which contain
+  # cmake helper files like FFTW3Config.cmake
+  find_package(FFTW3 CONFIG QUIET )
+  find_package(FFTW3f CONFIG QUIET )
+  if( FFTW3_FOUND AND FFTW3f_FOUND )
+    message(STATUS "FFTW3 found by using the FFTW3Config.cmake")
+    get_target_property(fftw3_libs FFTW3::fftw3 IMPORTED_LOCATION_RELEASE)
+    get_target_property(fftw3f_libs FFTW3::fftw3f IMPORTED_LOCATION_RELEASE)
+
+    set (FFTW_FOUND ${FFTW3_FOUND})
+    set (FFTW_INCLUDE_DIRS ${FFTW3_INCLUDE_DIRS})
+    set (FFTW_LIBRARIES ${fftw3_libs} )
+    list( APPEND FFTW_LIBRARIES ${fftw3f_libs} )
```

Does it need `get_target_property` here or would the following work

```suggestion
    set (FFTW_FOUND ${FFTW3_FOUND})
    set (FFTW_INCLUDE_DIRS ${FFTW3_INCLUDE_DIRS})
    set (FFTW_LIBRARIES FFTW3::fftw3)
    list( APPEND FFTW_LIBRARIES FFTW3:fftw3f )
```

---

I tried but get error:
ld.lld: error: unable to find library -lFFTW3:fftw3f

---

Leave as is then for now.